src/backends/backend.js

```diff
@@ -283,9 +309,9 @@ class Backend {
       .then(() => entryObj.slug);
   }
 
-  persistMedia(file) {
+  persistMedia(config, file) {
```

Comment:
Ok, will do. One note: I decided to pass `config` as the first argument because that is what `persistEntry` and `deleteEntry` currently do. Not sure if inconsistency between entry and media actions is a concern.

Reply:
I should have asked _why_ you did that - you're right, consistency first. Thanks for pointing that out.